src/pages/Home/Home.js

```diff
@@ -0,0 +1,39 @@
+import React, {Component} from 'react';
+
+export default class Home extends Component {
+
+    //This is a constructor
+    constructor(props) {
+
+        // This is allowing us to use any props that might have been passed from our
+        // parent
+        super(props);
+
+        // We set 3 states in this contructor (you can set more later) There is an empty
```

Comment:
LOVE all the comments, makes it super easy to understand what's going on/what we need to do